app/orm-adapters/node-orm2.js

```diff
@@ -87,22 +100,23 @@ export default class NodeORM2Adapter extends ORMAdapter {
     models.forEach((Model) => {
       let attributes = {};
       Model.eachAttribute((key, attribute) => {
-        attributes[key] = {
+        attributes[key] = assign({
           mapsTo: this.keyToColumn(key),
           type: this.denaliTypeToORMType(attribute.type)
-        };
+        }, attribute.options);
```

Comment:
This looks okay for now, but I'd like to be cautious about this going forward. The options hashes like the one handled here are one of the few areas that ORM adapters expose a public API, so this seems like a spot we want to put some thought into making sure we are okay supporting that.